src/components/WebCLI/utils.tsx

```diff
@@ -0,0 +1,42 @@
+import type { CommandHandler, ProcessOutput } from "./Output/types";
+
+/**
+ * Get the handler for the command, if any.
+ * @param command The command that was run.
+ * @param handlers The handlers to search through.
+ * @returns A matching handler, if it exists.
+ */
+const getHandler = <Handlers extends Record<string, CommandHandler>>(
```

Comment:
This logic is handled separately from the `processCommandOutput` function as it will be reused by the status table links.